Sources/ForageSDK/ForageSDK.swift

```diff
@@ -77,13 +77,13 @@ public class ForageSDK {
      Setup ForageSDK using Config struct.
      
      - Parameters:
-       - config: *Config* struct object to set environment.
+     - config: *Config* struct object to set environment.
      
-    ````
-      ForageSDK.setup(
-         ForageSDK.Config(environment: .sandbox)
-      )
-    ````
+     ````
+     ForageSDK.setup(
+     ForageSDK.Config(environment: .sandbox)
+     )
+     ````
      */
```

Comment:
looks like this was indented, but shouldn't have changed

Reply:
295f11bda61b011d3800308bb5c41eca996a82fd